packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -41,6 +41,7 @@ module.exports = {
   // This means they will be the "root" imports that are included in JS bundle.
   // The first two entry points enable "hot" CSS and auto-refreshes for JS.
   entry: [
+    'core-js/es6/symbol',
```

Comment:
I'd favor to add this polyfill to the general `polyfills` files instead of this array, esp. regarding maintenance. Would be good to have some kind of comment or a short description of why this is included.